extensions/terminal-suggest/src/terminalSuggestMain.test.ts

```diff
@@ -7,36 +7,39 @@ import { deepStrictEqual, strictEqual } from 'assert';
 import 'mocha';
 import { availableSpecs, getCompletionItemsFromSpecs } from './terminalSuggestMain';
 
-const availableCommands = ['cd', 'code', 'code-insiders'];
+const availableCommands = ['cd', 'code', 'code-insiders', 'c++', 'c.test', '.', '..', 'dir', 'ls'];
```

Comment:
While `.` is often a command, `..` shouldn't be. Should this be renamed or clarified somehow? Including `c.test` here is a bit confusing too